internal/pkg/base/base.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package policy
+package base
```

Comment:
suggestion: I still see `base` as a misleading name - can we think of a clearer one? If we're going to have many pluggable options, and this is just one, I'd rather not have a name that suggests (to me at least) that this is foundational (like a buttery biscuit base is foundational to a cheesecake... 😋) (i.e. that the others might be somehow dependent on it or be layered on top)

Naming is hard, but some ideas might be:
- 'rule based'
- 'attribute based/validation'
- 'property based/validation'
- ...

Just a thought.

Reply:
Yeah I understand where you are coming from. I would argue though that this is the base evaluator. All evaluators operate over CertificateRequestPolicies, and base is responsible for the default evaluations.

Reply:
In that case, would `default` be more accurate than `base`?

Reply:
Yeah true, just feel `default` suggests that it is optional or should be replaced with a non-default approver which it is not.

Not wedded to the name though :shrug:

Reply:
I think the main concern I have is that base suggests that other approvers might be layered on top - which is also not the case.

You're not keen on something like `attribute` or `property`?

Reply:
`attribute` works for me :+1:

Reply:
👍 thanks, I hope this will be clearer.